### ISSUE TITLE
ms edits

### ISSUE THREAD

**Issue by dcsuh** [dcsuh/temp_rsc_zoops#19]
- [x] clarify foraging language. use foraging and clearance. make sure clearance is about filtering and not about clearing infection. starts line 57
- [ ] check whether dnorm creates extra param (for fitting standard deviation) read dave's ecol letters 2013 paper to see if its similar
- [x] change model labels to model competition 1 (models 1A-1E) and model competition 2 (models 2A-E)
- [x] change figures to look better. model labels. legend. panel labels
- [x] make parameter table. 
- [x] parameter estimates and confidence intervals figure. try out figure with free axes. try out coefficients over temp or resource range
- [x] clarify language around additive model. avoid calling it a product. rewrite model so that effects are exponential sums
- [ ] consider more biological language in methods
- [ ] consistent language throughout ms
- [ ] condense langague/make things concise
- [ ] add more study system details

**Comment by dcsuh** [comment]
re: check whether dnorm creates extra param (for fitting standard deviation) read dave's ecol letters 2013 paper to see if its similar

pg. 130 of ben bolker book... normal distribution specifies mean and variance separately. In poisson and binomial distribution, variances is a fixed function of the mean. This means that mean and variance are separate parameters for normal distributions but are we estimating an extra parameter?

basically... normal distribution are defined by two parameters. mean and standard deviation. when we run the optimizer, are we estimating either of these parameters or are these just computed as constant features of the distribution?
